website/static/js/osfHelpers.js

```diff
@@ -269,6 +269,47 @@ var mapByProperty = function(list, attr) {
 
 
 /**
+  * Return CSS colors and percentage of a progress bar based on a value.
+  * Used to return a value for password complexity
+  */
+var valueProgressBar = function(value) {
+    if (value >= 0) {
+        if (value === 0) {
```

Comment:
You can refactor this to use a mapping.

```js
complexity_css_map = {
    0: {className: 'progress-bar ....', style: 'width: 20%'},
    1: ...
}
```

Also, better to use `className` rather than `class`, since `class` is a reserved word in newer version of JS.